pingcap/pd
# api: improve health API

### What problem does this PR solve? 
Closes #2152.

### What is changed and how it works?
This PR changes to check members' healthy by using healthy members instead of unhealthy members.

### Check List 

Tests 
 
 - Unit test
 - Manual test 
1. start three PDs and shut down one of them
2. use health API to check the health status
